docs/examples/index.rst

```diff
@@ -25,10 +25,13 @@ One of the best ways to get familiar with OPS is our tutorial, which has
 been used in several workshops and classrooms over the years. The main part
 of the tutorial (notebooks numbered 1-3) is essentially the same as the
 alanine dipeptide TPS example, and usually takes no more than 90 minutes to
-complete. It can be downloaded at the link below; see the readme file for
-details:
+complete. It can be downloaded at the link below or directly launched in
+Binder. Read the README file for details on using the tutorial.
 
-* https://gitlab.e-cam2020.eu/dwhswenson/ops_tutorial
+* https://github.com/openpathsampling/ops_tutorial |tutorial-binder|
+
+.. |tutorial-binder| image:: https://mybinder.org/badge_logo.svg
+   :target: https://mybinder.org/v2/gh/openpathsampling/ops_tutorial/HEAD?urlpath=lab
```

Comment:
is this one supposed to spin up into jupyter-lab? (It is neat, just making sure you done it by desing)

Reply:
Yes, because some of the advanced parts of the example are based on the CLI, and Lab gives you a terminal.

However, I think I still need to fix the binder env to handle widgets correctly in Lab.